libexec/klab-prove

```diff
@@ -48,7 +48,8 @@ K_OPTS=-Xmx10G \
       --no-alpha-renaming \
       --no-sort-collections \
       --output json \
-      --smt_prelude "$KLAB_OUT/prelude.smt2" \
+      --boundary-cells k,pc \
```

Comment:
Don't we need to treat internal `lemma`s differently to normal `behaviour`s here?

Reply:
@MrChico ^

Reply:
nope. The semantics here is that if both the RHS k and pc cells are matched by the current term, but the current term does not imply the target, we stop the execution. For internal specs, the k cell is matched by a lot of stuff, but not pc. For normal `behaviour`s the `pc` cell is matched by a lot of stuff, but not `k`.

Reply:
@MrChico got it.